product_brand/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2009 NetAndCo (<http://www.netandco.net>).
+# Copyright 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com>
+# Copyright 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com>
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2018 Daniel Campos <danielcampos@avanzosc.es>
+# Copyright 2018 Tecnativa - David Vidal
+# Copyright 2019 Giovanni - GSLabIt
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Brand Manager",
+    "version": "13.0.2.0.0",
+    "development_status": "Mature",
+    "category": "Product",
+    "summary": "Product Brand Manager",
+    "author": "NetAndCo, Akretion, Prisnet Telecommunications SA, "
+    "MONK Software, SerpentCS Pvt. Ltd., Tecnativa, Kaushal "
+    "Prajapati, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "license": "AGPL-3",
+    "depends": ["sale"],
```

Comment:
If you use `account.invoice.report` then you need `account` here as well.

Reply:
I think not necessary because the `payment` module has a dependency on `account` and `sale` module depend on `payment`.